application/templates/_helpers.tpl

```diff
@@ -36,13 +45,35 @@ Usage:
     {{- end }}
 {{- end -}}
 
-{{- define "application.namespace" -}}
-    
-        {{- if .Values.namespaceOverride }}
-            {{- .Values.namespaceOverride -}}
-        {{- else -}}
-            {{- .Release.Namespace -}}
-        {{- end -}}
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "application.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "application.labels" -}}
+helm.sh/chart: {{ include "application.chart" . }}
+app.kubernetes.io/name: {{ include "application.name" . }}
```

Comment:
this might conflict with argocd, since argocd adds the same label with value corrresponding to app name 
`app.kubernetes.io/name: argocd-application-name`

Reply:
it's a good point but the question if one isn't using argocd then what will happen? can you please read and check if this will become an issue with argocd?

Reply:
ok i can take a look into this